### ISSUE TITLE
Foreach in email template breaks layout (CommonMark)

### ISSUE THREAD

**Issue by jochensengier** [laravel/framework#31092]
- Laravel Version: 6.10.1
- PHP Version: 7.3.5
- Database Driver & Version: not applicable

### Description:
After switching to CommonMark under the hood in 6.10, using foreach in an e-mail template breaks the e-mail layout. This is not the same issue as reported in #31081 and #31065 . 

Tested with league/commonmark 1.1.2 & 1.2.0

```
@foreach (['name', 'email', 'telephone'] as $prop)
## {{ $prop }}
@component('mail::panel')
{{ $model->{$prop} }}
@endcomponent
@endforeach
```
![test1](https://user-images.githubusercontent.com/10118729/72136325-d7f5cb00-3388-11ea-9076-6be9dffcd341.jpg)

Adding an empty line before closing the foreach seems to fix the issue.

```
@foreach (['name', 'email', 'telephone'] as $prop)
## {{ $prop }}
@component('mail::panel')
{{ $model->{$prop} }}
@endcomponent

@endforeach
```
![test2](https://user-images.githubusercontent.com/10118729/72136333-de844280-3388-11ea-8de9-072cefc9fda0.jpg)

**Comment by taylorotwell** [comment]
After further testing this appears to be a bug in CommonMark.

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/463230/72159250-957cc000-3381-11ea-9436-85696f2dd2af.png">

**Comment by taylorotwell** [comment]
https://github.com/thephpleague/commonmark/issues/400

**Comment by jochensengier** [comment]
Thanks for taking the time to investigate the issue

**Comment by driesvints** [comment]
The default template within Laravel will be fixed in the next release: https://github.com/laravel/framework/commit/02a326ce5641b741fafb017f3d02b8917ba5e563

However, any existing exported views need to be updated manually. Please see the discussion in the linked issue above.

**Comment by jaythegeek** [comment]
I see that this has been closed, however I have noticed that this sometimes, randomly happens.

It has caused me to get this error in Horizon.

```Symfony\Component\Debug\Exception\FatalThrowableError: Class 'League\CommonMark\Environment' not found in /opt/bitnami/apache2/htdocs/vendor/laravel/framework/src/Illuminate/Mail/Markdown.php:103```

What is more strange is that it doesn't happen every time an email is sent out, not with any specific mailable, just every now and again it kicks back an error. I have upgraded Laravel to 6.12.0 but the problem still seems to be there.

So I suppose my question is do we need to make sure to respect the "blank line rule"  in our own mail.blade.php files?

Thanks for your help
